build.gradle

```diff
@@ -8,6 +8,7 @@ buildscript {
     repositories {
         mavenCentral()
         maven { url "https://oss.sonatype.org/content/repositories/snapshots/" }
+        maven { url "http://maven.imagej.net/content/repositories/public/" }
```

Comment:
The reason I used that [Maven repo](http://maven.imagej.net/content/repositories/public/) is because I generally use this [site](https://mvnrepository.com/artifact/com.github.WilliamAHartman/Jamepad/1.2) for looking up dependencies, and it said the dependency was listed in that repo.  

Sorry, I see now where you got the reference to [jitpack](https://github.com/williamahartman/Jamepad/#getting-jamepad).